nuxeo-features/nuxeo-platform-oauth/src/main/java/org/nuxeo/ecm/platform/oauth2/request/AuthorizationRequest.java

```diff
@@ -172,45 +209,36 @@ public String getAuthorizationKey() {
         return authorizationKey;
     }
 
-    private static void deleteExpiredRequests() {
```

Comment:
Shouldn't this be implemented with a TTL of 10 minutes for the `authorizationRequestStore` transient store?